pkg/vm/engine/disttae/logtailreplay/partition_state.go

```diff
@@ -138,17 +138,21 @@ func NewPartitionState(noData bool) *PartitionState {
 		rows:         btree.NewBTreeGOptions((RowEntry).Less, opts),
 		blocks:       btree.NewBTreeGOptions((BlockEntry).Less, opts),
 		primaryIndex: btree.NewBTreeGOptions((*PrimaryIndexEntry).Less, opts),
-		dirtyRows:    btree.NewBTreeGOptions((RowEntry).Less, opts),
+		dirtyBlks:    btree.NewBTreeGOptions((BlockEntry).Less, opts),
 	}
 }
 
+func (p *PartitionState) GetDirtyBlks() *btree.BTreeG[BlockEntry] {
+	return p.dirtyBlks
```

Comment:
Don't expose private fields since the btree implementation may change. And this field must not be mutated, exposing it may break this assumption.